docs/docs/data-structures/tree/schema-evolution/index.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: Schema Evolution
+sidebar_position: 8
+---
+
+# Schema Evolution
+
+As your application grows and evolves, you may need to modify the structure of your [SharedTree](../index.mdx) data.
+SharedTree makes it possible to update your [schemas](../schema-definition.mdx) in ways that allow an app to load or upgrade documents created or edited by earlier versions of the app (see [understanding compatibility](./index.mdx#understanding-compatibility)).
+
+> **Note:** For detailed information about which types of changes are safe versus breaking, see [types of schema changes](./types-of-changes).
+
+## Stored vs View Schema
+
+When updating your schema, it's important to understand the difference between the two types of schemas:
+
+- **View Schema**: The schema defined in your application code using [`SchemaFactory`](../../../api/fluid-framework/schemafactory-class)
+- **Stored Schema**: The schema persisted with each document. It describes what type of tree can be stored in the document.
+
+When a document is first created, its stored schema is set using the given view schema.
+Afterwards, it is immutable until explicitly [upgraded](./index.mdx#schema-upgrade-process).
+
+### Stored Schema Invariants
+
+The stored schema follows additional constraints that don't apply to view schemas:
+
+**1. Complete Description**
+The stored schema completely describes all data in the tree.
+Every node and field in the document is known to the stored schema.
+
+**2. Additive Only**
+The stored schema can grow over time but cannot shrink:
+- New allowed types can be added to fields
+- Existing types cannot be removed from fields
+- Fields cannot be removed from node types
+- Node types cannot be completely removed
```

Comment:
This is potentially useful information for someone trying to deeply understand how the system works, but is probably unhelpful and confusing to someone trying to learn how to do a schema migration for the first time. Node types cannot be _completely_ removed? What does that mean - can they remove nodes? Or not? Where? When?

Maybe instead, say something like "All nodes types present in the stored schema will exist forever, even if they stop being used by the view schema"